lib/wasix/src/runners/wasi.rs

```diff
@@ -353,6 +353,10 @@ impl crate::runners::Runner for WasiRunner {
             }
         }
 
+        if let Some(cwd) = cmd.metadata().annotation::<String>("cwd")? {
```

Comment:
Also, shouldn't we add this to the type definitions in webc?

Reply:
It was added as a custom annotation which does not have a type definition and is just a plain `Value`, but adding it as a wasi annotation will require a change to webc.